test/test_query2.cpp

```diff
@@ -6056,44 +6056,93 @@ TEST(Query_FullTextMulti)
              "scientifiques de niveau recherche, publiés ou non, émanant des établissements d’enseignement et de "
              "recherche français ou étrangers, des laboratoires publics ou privés.");
     table->create_object().set(col, "object object object object object duplicates");
+    table->create_object().set(col, "one two three");
+    table->create_object().set(col, "three two one");
+    table->create_object().set(col, "two one");
+
+    // object:      0, 1, 2, 4, 6
+    // objects:     0, 1, 3
```

Comment:
```suggestion
    // objects:     0, 1, 3
    // object-oriented: 0, 1
    // table-oriented: 0
```

Reply:
We can choose to accept '-' if it is embedded in a word. But then we will not find just "object" in "object-oriented". Is that what we want?

Reply:
It's something for whole new project it seems. Not on the plate for now.